src/App.jsx

```diff
@@ -1,35 +1,22 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import "./App.css";
+import Header from "./components/Header";
+import ItemContainer from "./components/ItemContainer";
+import { useState } from "react";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [search, setSearch] = useState("");
+  const [articles, setArticles] = useState([]);
```

Comment:
This state I would argue is way too high, what other component needs access to this data?